uwblueprint/sistering
# Update seed, fix tag overflow issues

## Ticket link

Closes #623, #624



## Implementation description
* make prod seed data only contain skills, branches, and languages
* added new branches to the seed data
* created an overflowntext component to address overflow issues with long branch names


## Steps to test
1. Ensure the seed data works correctly
2. Ensure overflow behaves correctly in the following places

<img width="1001" alt="Screen Shot 2022-08-22 at 6 40 21 PM" src="https://user-images.githubusercontent.com/82241706/186052472-f5eab208-0424-4590-875f-9ec59f7c4801.png">
<img width="500" alt="Screen Shot 2022-08-22 at 6 40 46 PM" src="https://user-images.githubusercontent.com/82241706/186052485-2037c6b3-a3e0-4c6e-944e-a68ad237e209.png">
<img width="497" alt="Screen Shot 2022-08-22 at 6 35 31 PM" src="https://user-images.githubusercontent.com/82241706/186052514-a6a92d72-fe3f-4348-a470-dd56c433020b.png">



## What should reviewers focus on?
* addresses existing bugs, no new bugs are produced

## Checklist
